README.md

```diff
@@ -38,39 +38,43 @@ sha1: 50bebedad9e0f10746a52b07239e47fa6c284d03 (Alternate)
 
 Congratulations, you are now sailing with the Ship of Harkinian! Have fun!
 
-## Additional Setup Information
+## About Versions
+
+The Ship of Harkinian's versioning system is a unique system, comprising of a three-numbered version (`x.y.z`) and a version codename.  The main version number `x` signifies major feature updates that require a new `oot.otr` file.  The version number `y` signifies major feature updates and bugfixes that most likely do not require a new `oot.otr` file, although it may be a good idea to create a new one.  The version number `z` signifies minor updates such as a collection of bugfixes.
 
-The Ship of Harkinian uses a proprietary versioning system consisting of a sci-fi film character followed by a phonetic alphabet code word. The film character represents a major release version which increments with the addition of many new features and bug fixes. The code word represents a minor release version which increments with small updates mainly comprised of bug fixes. For example, `DECKARD ALFA`.
+The version codename consists of a sci-fi film character followed by a phonetic alphabet code word. The film character represents a major release version which increments with the major `x` or `y` version bumps. The code word represents a minor release version which increments with the minor `z` version bump.
+
+## Additional Setup Information
 
 ### Windows Rom Extraction
 
-* Open OTRGui.exe, and select one of the supported roms listed above, to generate the `oot.otr` archive file.
+* Open `OTRGui.exe` and click on "Open OOT ROM", then select your supported copy of the game (see section "Supported Games").
 * If a second button already exits then `oot.otr` already exists. To prevent overwriting the old `oot.otr` use this button to choose a new game directory. The new directory must not already contain an `oot.otr` to prevent an error.
-* When the process completes, place `oot.otr` beside `soh.exe` if it is not already.
+* When the process completes, you should have an `oot.otr` or `oot-mq.otr` file in your main directory, depending on the version of the game used to build it.
```

Comment:
Might be good to add `or both if you want to play randomizer with vanilla and MQ dungeons shuffled together` or something along those lines.

Reply:
I'll add a note that suggests you can have both. Good call!